### DISCUSSION TITLE
Why ignoring routes without name?

### DISCUSSION THREAD

**Discussion by cnmuc** [gtm-support/vue-gtm#232] [Q&A]
Just found that routes without a name are ignored.

![image](https://user-images.githubusercontent.com/4402945/180410298-751b162a-3467-4bb7-b800-054c9fa04a34.png)

What's the reason for this?

**Comment by Shinigami92** [comment]
I'm not sure. I assume you want to check it via the `ignoredViews` function passed as option?
I'm open for PR.
